vulyk/models/user.py

```diff
@@ -70,6 +70,21 @@ def is_eligible_for(self, task_type):
 
         return task_type in chain(*(g.allowed_types for g in self.groups))
 
+    @classmethod
+    def pre_save(cls, sender, document, **kwargs):
+        """
+        :type sender: type
+        :type document: User
+        :type kwargs: dict
+        """
+        if all(map(lambda x: x.id != "default", document.groups)):
+            try:
+                document.groups = [Group.objects.get(id="default")]
+            except Group.DoesNotExist:
+                raise Group.DoesNotExist("Please run 'manage.py init ...'")
```

Comment:
And this will shit 500 to the first user who will try to register.

What if we'll hide login/register button if default group isn't available and show some message about manage.py init instead